docs/design/pages/rule-list/features/toggle-rule-active/e2e-test-strategy.md

```
@@ -0,0 +1,122 @@
+## 網羅性チェック
+
+- [x] 基本操作（true→false、false→true）
+- [x] DB永続化（リロード後の状態維持）
+- [x] 複数データの独立性
+- [x] コンソールエラー監視
+- [ ] エラーハンドリング → 結合テストでカバー
+- [ ] 部分的成功 → 結合テストでカバー
+
+## テストファイル構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)